andela/ah-jawans-frontend
# #166790065:  bookmark article for reading later

#### What does this PR do?
This PR Supports Bookmarking articles for reading later.

#### Description of Task to be completed?
Bookmark articles for reading later when you view one article.

#### How should this be manually tested?
This can be tested by cloning the repo, run npm install and then run the command npm run dev, and then check by opening reading article and click bookmark image.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166790065
	
#### Screenshots (if appropriate)
![rKFulN14oA](https://user-images.githubusercontent.com/51748364/64815781-a3266280-d5a6-11e9-83af-db57947eed32.gif)
